RevenueCatUI/UIKit/PaywallViewController.swift

```diff
@@ -32,18 +32,37 @@ public class PaywallViewController: UIViewController {
     }
 
     private let offering: Offering?
+    private let fonts: PaywallFontProvider
     private let displayCloseButton: Bool
 
     /// Initialize a `PaywallViewController` with an optional `Offering`.
     /// - Parameter offering: The `Offering` containing the desired `PaywallData` to display.
     /// `Offerings.current` will be used by default.
     /// - Parameter displayCloseButton: Set this to `true` to automatically include a close button.
     @objc
+    public convenience init(
+        offering: Offering? = nil,
+        displayCloseButton: Bool = false
+    ) {
+        self.init(
+            offering: offering,
+            fonts: DefaultPaywallFontProvider(),
+            displayCloseButton: displayCloseButton
+        )
+    }
+
+    /// Initialize a `PaywallViewController` with an optional `Offering` and ``PaywallFontProvider``.
+    /// - Parameter offering: The `Offering` containing the desired `PaywallData` to display.
+    /// `Offerings.current` will be used by default.
+    /// - Parameter fonts: An optional ``PaywallFontProvider``.
+    /// - Parameter displayCloseButton: Set this to `true` to automatically include a close button.
     public init(
         offering: Offering? = nil,
+        fonts: PaywallFontProvider,
```

Comment:
Same here about a non-optional parameter after an optional one. We should also make sure these work well in objc I think.

Reply:
This is Swift only because `PaywallFontProvider` can't be represented in Obj-C.

Reply:
Ohh ok... yeah we probably need to figure out a way to do this later in the hybrids anyway, so this works 👍

Reply:
Yeah we'll need to wrap the type somehow. The font parameter is a SwiftUI type.